packages/toolkit/src/view/knowledge/KnowledgeBaseView.tsx

```diff
@@ -201,48 +254,60 @@ export const KnowledgeBaseView = (props: KnowledgeBaseViewProps) => {
 
   return (
     <div className="h-screen w-full bg-semantic-bg-alt-primary">
-      {showDeleteMessage && knowledgeBaseToDelete && (
-        <DeleteKnowledgeBaseNotification
-          knowledgeBaseName={knowledgeBaseToDelete.name}
-          handleCloseDeleteMessage={handleCloseDeleteMessage}
-          undoDelete={undoDelete}
-        />
-      )}
       {showCreditUsage && !isLocalEnvironment && (
         <CreditUsageFileNotification
           handleCloseCreditUsageMessage={handleCloseCreditUsageMessage}
           fileName="test"
         />
       )}
       <div className="grid w-full grid-cols-12 gap-6 px-8">
-        <div className="pt-20 sm:col-span-4 md:col-span-3 lg:col-span-2">
-          <Sidebar
-            activeTab={activeTab}
-            onTabChange={handleTabChange}
-            selectedKnowledgeBase={selectedKnowledgeBase}
-            onDeselectKnowledgeBase={handleDeselectKnowledgeBase}
-          />
-        </div>
-        <div className={`sm:col-span-8 md:col-span-9 lg:col-span-10 pt-5`}>
+        {selectedKnowledgeBase && (
+          <div className="pt-20 sm:col-span-4 md:col-span-3 lg:col-span-2">
+            <Sidebar
+              activeTab={activeTab}
+              onTabChange={handleTabChangeAttempt}
+              selectedKnowledgeBase={selectedKnowledgeBase}
+              onDeselectKnowledgeBase={handleDeselectKnowledgeBase}
+            />
+          </div>
+        )}
+        <div
+          className={`${selectedKnowledgeBase
+            ? "sm:col-span-8 md:col-span-9 lg:col-span-10"
+            : "col-span-12"
+            } pt-5`}
+        >
           {activeTab === "catalogs" && (
             <KnowledgeBaseTab
               onKnowledgeBaseSelect={handleKnowledgeBaseSelect}
               onDeleteKnowledgeBase={handleDeleteKnowledgeBase}
               accessToken={props.accessToken}
-              pendingDeletions={pendingDeletions}
-            />
-          )}
-          {activeTab === "upload" && selectedKnowledgeBase && (
-            <UploadExploreTab
-              knowledgeBase={selectedKnowledgeBase}
-              onProcessFile={handleProcessFile}
-              onTabChange={handleTabChange}
+              knowledgeBases={knowledgeBases.data || []}
+              knowledgeBaseLimit={knowledgeBaseLimit}
+              namespaceType={namespaceType}
+              subscription={subscription}
```

Comment:
@thewbuk Let's stop using shortage and use ternary instead

`activeTab === "catalogs" ? ... : null`